stackrox/acs-fleet-manager
# Use unique namespace names for central deployments

## Description
Use unique namespace names for central deployments.

## Checklist (Definition of Done)


## Test manual
1. Add yourself in the `quota-management-list-configuration.yaml`
2. Create a few instances using public API
3. Make sure that unique namespaces were created by the fleetshard

```
# To run tests locally run:
make db/teardown db/setup db/migrate
make ocm/setup OCM_OFFLINE_TOKEN=<ocm-offline-token> OCM_ENV=development
make verify lint binary test test/integration
```
